utils/src/stable_buf.rs

```diff
@@ -1,25 +1,9 @@
-// The contents of this file are based on https://github.com/tokio-rs/tokio-uring at commit 7761222.
```

Comment:
Can you add a note (under a `# Acknowledgements` heading) this was inspired by this crate?

Example: https://github.com/commonwarexyz/monorepo/blob/d824eca53d25f3b874159ac8afd432bd0f31b100/cryptography/src/bls12381/primitives/mod.rs#L3-L10